.github/workflows/release-desktop-mas.yml

```
@@ -22,36 +22,25 @@ jobs:
         with:
           lfs: true
 
-      - name: Dotenv Action
-        id: dotenv
-        uses: OneKeyHQ/actions/dotenv-action@main
+      - name: Run Shared Env Setup  
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)